org_status/status_providers/__init__.py

```diff
@@ -26,10 +26,10 @@ def __init__(self, group):
 
         self._group = group
         self._group_url = self.BadgeTemplate.format(
-          group=group, repo='{repo}', branch='{branch}')
+          group=group, repo='{repo}', branch='{branch}', host='{host}')
 
-    def get_badge_url(self, repo, branch='master'):
-        return self._group_url.format(repo=repo, branch=branch)
+    def get_badge_url(self, repo, host, branch='master'):
```

Comment:
If `1`  of https://github.com/ksdme/org-status/pull/19#discussion_r230565472 is to be implemented then `host` should be passed in the initialiser .